values.yaml

```diff
@@ -1257,6 +1257,10 @@ concourse:
       ##
       heartbeatInterval: 30s
 
+      ## How long to wait for requests to Garden to complete. 0 means no timeout.
+      ## default: 5m
```

Comment:
nit: Would just drop the the example value to be consistent with the 2 other entries for this

Reply:
Done, thanks!